packages/client/navigation/index.tsx

```diff
@@ -198,6 +199,15 @@ export const RootNavigator: React.FC<ColorModeProps> = ({ colorScheme }) => {
                 component={UserProfileNavigator}
             />
 
+            <RootStack.Screen
```

Comment:
Did you not created any navigator to be able to just cut and paste the screen in the upcoming myProfileNavigator ?

Reply:
Yes, I did not want to take a decision in this PR for a simple screen.